test/studies/shootout/chameneos-redux/elliot/chameneosredux-ejr.chpl

```diff
@@ -0,0 +1,291 @@
+/* The Computer Language Benchmarks Game
+   http://benchmarksgame.alioth.debian.org/
+
+   contributed by Hannah Hemmaplardh, Lydia Duncan, and Brad Chamberlain
+   derived in part from the GNU C version by Dmitry Vyukov
+*/
+
+config const n = 600,              // number of meetings (must be >= 0)
+             popSize1 = 3,         // size of population 1 (must be > 1)
+             popSize2 = 10;        // size of population 2 (must be > 1)
+
+enum Color {blue=0, red, yellow};  // the chameneos colors
+use Color;                         // permit unqualified references to them
+
+
+//
+// Print the color equations and simulate the two population sizes.
+//
+proc main() {
+  printColorEquations();
+
+  const group1 = new Population(popSize1);
+  const group2 = new Population(popSize2);
+
+  cobegin {
+    group1.holdMeetings(n);
+    group2.holdMeetings(n);
+  }
+
+  group1.printColors();
```

Comment:
Mentioned this in person, but for the record:

The [expected output](http://benchmarksgame.alioth.debian.org/download/chameneosredux-output.txt) is
```
  // group1 initial colors

  // group1 output

  // group2 initial colors

  // group2 output 
```

```
- write the colours the creatures start with
- after N meetings have taken place, for each creature write the number of creatures met and spell out the number of times the creature met a creature with the same name (should be zero)
```
so you can't just print the colors before holding meetings. Technically you could print group 1's meetings before the cobegin, but that looks strange IMO